### ISSUE TITLE
Crostini: required plugin "com.google.gct.login" not installed.

### ISSUE THREAD

**Issue by Splaktar** [GoogleCloudPlatform/google-cloud-intellij#2295]
- Cloud Tools for IntelliJ version: `v16.11.6`
- Google Cloud SDK version _(run `gcloud version` on the command line)_: `226.0.0`
- OS: Chrome OS `70.0.3538.76 (Official Build) (64-bit)`

**What did you do?**
Launched IntelliJ IDEA with the Google Cloud Tools plugin.

**What did you expect to see?**
The plugin enabled and working like I see on my macOS device.

**What did you see instead?**
```
Plugin Error
Problems found loading plugins:
Plugin "Google Cloud Tools" was not loaded: required plugin "com.google.gct.login" not installed.
```			

I am not able to get far enough to see https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/2166 or https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/2210.

I see the same error with WebStorm, but it also gives me an error related to a missing Gradle plugin which doesn't appear to be available to WebStorm on Chrome OS.

**Comment by etanshaul** [comment]
Hi @splaktar. Thanks for the report. Are you using IDEA version 2018.3? If so, we are aware of the issue and will release a new version of the plugin to fix this in the next day or so. This is happening because the plugin does not properly support this version.

See this thread (with a temporary workaround): https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/2287#issuecomment-441183232

**Comment by Splaktar** [comment]
Yep, using the latest of both. Thank you for the link and workaround!

**Comment by etanshaul** [comment]
@Splaktar, as updated on #2287 this issue should be fixed with the latest release of the plugin: 18.11.1

Closing this issue for now. Thanks.

**Comment by Splaktar** [comment]
I can confirm that the 18.11.1 update resolved this. Thank you!
